src/NUnitEngine/nunit.engine.netstandard/nunit.engine.netstandard.csproj

```diff
@@ -0,0 +1,174 @@
+      <Link>Internal\CecilExtensions.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\DirectoryFinder.cs">
+      <Link>Internal\DirectoryFinder.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\Logging\InternalTrace.cs">
+      <Link>Internal\Logging\InternalTrace.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\Logging\InternalTraceWriter.cs">
+      <Link>Internal\Logging\InternalTraceWriter.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\Logging\Logger.cs">
+      <Link>Internal\Logging\Logger.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\PathUtils.cs">
+      <Link>Internal\PathUtils.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\ResultHelper.cs">
+      <Link>Internal\ResultHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\SettingsGroup.cs">
+      <Link>Internal\SettingsGroup.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Internal\SettingsStore.cs">
+      <Link>Internal\SettingsStore.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\ITestEngineRunner.cs">
+      <Link>ITestEngineRunner.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\ITestRunnerFactory.cs">
+      <Link>ITestRunnerFactory.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Properties\AssemblyInfo.cs" Link="Properties\AssemblyInfo.cs" />
+    <Compile Include="..\nunit.engine\Runners\AbstractTestRunner.cs">
+      <Link>Runners\AbstractTestRunner.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Runners\AggregatingTestRunner.cs">
+      <Link>Runners\AggregatingTestRunner.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Runners\ITestExecutionTask.cs">
+      <Link>Runners\ITestExecutionTask.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Runners\TestEventDispatcher.cs">
+      <Link>Runners\TestEventDispatcher.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Runners\TestExecutionTask.cs">
+      <Link>Runners\TestExecutionTask.cs</Link>
+    </Compile>
+    <Compile Include="..\nunit.engine\Services\ResultService.cs" Link="Services\ResultService.cs" />
+    <Compile Include="..\nunit.engine\Services\ResultWriters\NUnit3XmlResultWriter.cs" Link="Services\ResultWriters\NUnit3XmlResultWriter.cs" />
+    <Compile Include="..\nunit.engine\Services\ResultWriters\TestCaseResultWriter.cs" Link="Services\ResultWriters\TestCaseResultWriter.cs" />
+    <Compile Include="..\nunit.engine\Services\TestFilterBuilder.cs" Link="Services\TestFilterBuilder.cs" />
+    <Compile Include="..\nunit.engine\Services\TestFilterService.cs" Link="Services\TestFilterService.cs" />
+    <Compile Include="..\nunit.engine\Services\TestSelectionParser.cs" Link="Services\TestSelectionParser.cs" />
+    <Compile Include="..\nunit.engine\Services\Tokenizer.cs" Link="Services\Tokenizer.cs" />
```

Comment:
Seems like a glob could take care of almost all these items linked from the `..\nunit.engine` folder. That's one less csproj to constantly be touching. :D (I can not wait until we do this with all the projects!)